lms/djangoapps/teams/tests/test_views.py

```diff
@@ -1299,12 +1300,18 @@ def setUp(self):  # pylint: disable=arguments-differ
         ('student_enrolled', 403),
         ('staff', 204),
         ('course_staff', 204),
-        ('community_ta', 204)
+        ('community_ta', 204),
+        ('admin', 204)
     )
     @ddt.unpack
     def test_access(self, user, status):
+        if user is not None:
+            team_list = self.get_teams_list(user='course_staff', expected_status=200)
+            previous_count = team_list['count']
         self.delete_team(self.solar_team.team_id, status, user=user)
         if status == 204:
+            team_list = self.get_teams_list(user='course_staff', expected_status=200)
```

Comment:
or you can just do
- Delete the team `self.delete_team(self.solar_team.team_id, status, user=user)`
- If the team is deleted successfully, 
- Check if the team id no longer exist when you get a `team_list`

Reply:
Okay lemme do that. 👍